easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.1.9.eb

```diff
@@ -46,13 +46,12 @@ exts_list = [
         'source_urls': ['https://pypi.python.org/packages/source/v/vsc-install'],
         'checksums': ['f1d0796aa6201c0d7e1c6bde591153be1b8bde2f04860494272432c1d30f18f5'],
     }),
-    ('vsc-base', '2.9.2', {
+    ('vsc-base', '2.9.6', {
         'modulename': 'vsc.utils',
         'source_urls': ['https://pypi.python.org/packages/source/v/vsc-base'],
-        'checksums': ['55803ce00f29bc0eb8b4e37518b21368b4c0f4cdb37c3a168912fc815fd88f15'],
+        'checksums': ['a62158022c36b5b456771ee8145d9ea3d299b301240e9724d04052e6aa0400b3'],
     }),
     (name, version, {
-        # don't check for import of vsc.mympirun, since that may fail if vsc-base is picked up from the OS
```

Comment:
@smoors please restore this comment?

Reply:
@boegel I actually did not delete it, it was `eb --inject-checksums` who did it :)